contracts/vaults/ERC20RootVault.sol

```diff
@@ -17,6 +17,7 @@ contract ERC20RootVault is IERC20RootVault, ERC20Token, ReentrancyGuard, Aggrega
     using SafeERC20 for IERC20;
     using EnumerableSet for EnumerableSet.AddressSet;
 
+    bytes4 public constant REBALANCE_SELECTOR = 0xe78eeb6d;
```

Comment:
Let's make it more explicit
+ I would name it depositCallback + withdrawCallback